### ISSUE TITLE
Cannot set maxHeight using vH

### ISSUE THREAD

**Issue by lukesmurray** [davidjbradshaw/iframe-resizer#930]
**Describe the bug**
Cannot set maxHeight option to `vH`

**To Reproduce**

Set the maxHeight option to 100vh.  When you look at the iframe you'll see the style tag for the iframe does not have a maxHeight set.

**Expected behavior**
I would expect to be able to set the iframe height to units other than px. vH is incredibly useful since it is automatically responsive if the user changes the screen size.

**Additional context**

```
      function addStyle(style) {
        if (
          Infinity !== settings[iframeId][style] &&
          0 !== settings[iframeId][style]
        ) {
          iframe.style[style] = settings[iframeId][style] + 'px'
          log(
            iframeId,
            'Set ' + style + ' = ' + settings[iframeId][style] + 'px'
          )
        }
      }
```

I believe the issue is due to the addStyle function add a `px` suffix to the iframe styles.
Not sure how to fix while retaining backwards compatibility.

Obviously these same issues would apply to `vw` as well if my reasoning is correct.

**Comment by davidjbradshaw** [comment]
I guess we should do `isNumber()` and only add `px` if true. 

Would welcome a PR

**Comment by lukesmurray** [comment]
That sounds reasonable and was similar to what I was thinking. I'll use lodash to check for isNumber since that's already a dependency of the lib.

**Comment by davidjbradshaw** [comment]
Can you just cut and paste the function, the built code has no dependencies in it or we can just write it
```js
function isNumber(value) {
  return typeOf value === ‘Number’
}

**Comment by davidjbradshaw** [comment]
Fixed in v4.3.2.
